go/skaffold.yaml

```diff
@@ -38,46 +38,6 @@ profiles:
   # integration: set of `skaffold debug`-like integration tests
   - name: integration
     patches:
-      - op: add
-        path: /build/artifacts/-
-        value:
-          image: go113app
-          context: test/goapp
-          docker:
-            buildArgs:
-              GOVERSION: 1.13
-      - op: add
-        path: /build/artifacts/-
-        value:
-          image: go114app
-          context: test/goapp
-          docker:
-            buildArgs:
-              GOVERSION: 1.14
-      - op: add
-        path: /build/artifacts/-
-        value:
-          image: go115app
-          context: test/goapp
-          docker:
-            buildArgs:
-              GOVERSION: 1.15
-      - op: add
-        path: /build/artifacts/-
-        value:
-          image: go116app
-          context: test/goapp
-          docker:
-            buildArgs:
-              GOVERSION: 1.16
-      - op: add
-        path: /build/artifacts/-
-        value:
-          image: go117app
-          context: test/goapp
-          docker:
-            buildArgs:
-              GOVERSION: 1.17
```

Comment:
I'd suggest backing out this change and just keep this PR focussed on making the Go tests pass (and drop the Python `--DEBUG` too)

Reply:
Yep, I was trying exactly that, but I got the error related with [no space left on device](https://github.com/GoogleContainerTools/container-debug-support/actions/runs/5524045500/jobs/10075802831) :/ (same was for the `--DEBUG` flag). I can revert the changes as you are mentioning, but we'll need to merge this PR with broken tests, WDYT?

Reply:
Ok, just make it clear in the commit that we're fixing all the things :-)

Reply:
Done! Every change is in a different commit and the PR description has the changes :)